src/luos.c

```diff
@@ -60,6 +60,20 @@ void Luos_Init(void)
     service_number = 0;
     memset(&luos_stats.unmap[0], 0, sizeof(luos_stats_t));
     Robus_Init(&luos_stats.memory);
+
+    uint16_t package_index = 0;
+    if (package_number)
+    {
+        while (package_index < package_number)
+        {
+            package_table[package_index].Init();
+            package_index += 1;
+        }
+    }
+    else
+    {
+        Luos_EmptyNode();
+    }
```

Comment:
Is those modifications have any relation with self-test?

Reply:
Commits on luos.c had been dropped, those changes were a mistake.